vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -85,16 +85,17 @@ public void scrollOverlay_morePagesLoaded() {
         assertLoadedItemsCount(
                 "There should be 150 items after loading three pages", 150,
                 stringBox);
-        assertRendered("Item 115");
+        assertRendered("Item 102");
     }
 
     @Test
     public void openPopup_scrollToEnd_onlyFirstAndLastPagesLoaded() {
```

Comment:
The test name should be updated (remove first page).

Reply:
Done